### ISSUE TITLE
Content Blocks does not work on Member Type

### ISSUE THREAD

**Issue by Wade-Moerke** [leekelleher/umbraco-contentment#380]
### Which Contentment version are you using?

4.6.0

### Which Umbraco version are you using? For example: 10.3.2 - don't just write v10

13.0.3

### Bug summary

When there is a Content Blocks data type on a Member type, an error occurs when trying to add a new content block to that picker. I click on "Add content" on a member and the following error occurs: "No content with that id. (Parameter 'parentId')".  I have attached screenshots with the actual errors and browser console errors.  It seems to me like it is trying to find a parent id of a member, but it is only searching in content, so it is unable to find it.

![image](https://github.com/leekelleher/umbraco-contentment/assets/103143527/623cbf4e-acd4-48da-b7fa-525cd636e9bf)

![image](https://github.com/leekelleher/umbraco-contentment/assets/103143527/a8445b05-a918-4a82-a4bd-897f1419f93e)

Stack trace:
```
at Umbraco.Cms.Core.Services.ContentService.Create(String name, Int32 parentId, IContentType contentType, Int32 userId)
   at Umbraco.Cms.Core.Services.ContentService.Create(String name, Int32 parentId, String contentTypeAlias, Int32 userId)
   at Umbraco.Cms.Web.BackOffice.Controllers.ContentController.GetEmptyInner(IContentType contentType, Int32 parentId)
   at lambda_method1011(Closure, Object, Object[])
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|26_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
```

![image](https://github.com/leekelleher/umbraco-contentment/assets/103143527/02c70682-cddc-4510-b3b8-4338ab84a8bf)

### Steps to reproduce

Add a Content Blocks picker to a member, then go into the member and try to add content through the picker.

### Expected result / actual result

Content Blocks should work the same on a member as it works on a document type.

### Do you have Umbraco ModelsBuilder enabled?

- [X] Yes, it is enabled.

### What browsers are you seeing the problem on?

Chrome

**Comment by leekelleher** [comment]
@Wade-Moerke Thanks for reporting this. I haven't tried the Content Blocks editor on a Member type before (I honestly never thought it'd be a use case). I'll give it a test tomorrow, see what can be fixed.

**Comment by leekelleher** [comment]
Update on this, I can reproduce it - from what I can tell Content Blocks never worked in the Media or Member sections.
Good news is that I've got a fix, which I'll try to get ready for a patch release within the next week. _(I'm juggling various things at the moment 🤹)_

**Comment by Wade-Moerke** [comment]
Sounds good, thanks for the updates!

**Comment by Wade-Moerke** [comment]
@leekelleher Just checking in on the status of this.  I have a few things I would really like to use this functionality for instead of trying to figure out a different way of doing things.

**Comment by Wade-Moerke** [comment]
@leekelleher Just checking in on the status of this again.  I'm still hoping I can make use of this with the fix.

**Comment by leekelleher** [comment]
@Wade-Moerke I've released [Contentment v4.6.1](https://github.com/leekelleher/umbraco-contentment/releases/tag/4.6.1) that contains the patch for Content Blocks in the Members (and Media) section.
